vendor/ddcloud/resource_additional_nics.go

```diff
@@ -0,0 +1,75 @@
+package ddcloud
+
+import "github.com/hashicorp/terraform/helper/schema"
+
+const (
+	resourceKeyNicServerID     = "serverid"
+	resourceKeyNicID           = "id"
+	resourceKeyNicVLANID       = "vlan_id"
```

Comment:
Could you change `serverid` to just `server`? That way it's the same as `ddcloud_server.networkdomain` (etc).